index.html

```diff
@@ -0,0 +1,225 @@
+            <li class="navbar-links">
+              <a href="index.html">Home</a>
+            </li>
+            <li class="navbar-links">
+              <a href="#program_section">Programs</a>
+            </li>
+            <li class="navbar-links">
+              <a href="#speakers_section">Speakers</a>
+            </li>
+            <li class="navbar-links">
+              <a href="#partners">Sponsor</a>
+            </li>
+            <li class="navbar-links">
+              <a href="about.html">About</a>
+            </li>
+              <li class="navbar-links campaign_link">
+                <a href="#contact"> CC Campaign</a>
+              </li>
+          </ul>
+          <div class="mobile-menu-bars mobile" id="menu-bars">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+        </div>
+      </nav>
+    <!-- navbar section end  -->
+    <!-- navbar for mobile start  -->
+      <nav class="overlay overlay-slide-left" id="overlay">
+        <ul class="mobile-nav-items">
+        <li id="nav-1"><a href="#home" class="nav-mobile-links">Home</a></li>
+        <li id="nav-1"><a href="#program_section" class="nav-mobile-links">Programs</a></li>
+        <li id="nav-2"><a href="#speakers_section" class="nav-mobile-links">Speakers</a></li>
+        <li id="nav-3"><a href="#partners" class="nav-mobile-links">Sponsor</a></li>
+        <li id="nav-3"><a href="about.html" class="nav-mobile-links">About</a></li>
+        <li id="nav-3"><a href="#" class="nav-mobile-links">Campaign</a></li>
+        </ul>
+      </nav>
+      <!-- navbar for mobile end  -->
+    </header>
+      <!-- main header start -->
+    <section class="header_part" id="home">
+
+      <div class="header_part_main_div">
+        <h2 class="hello-text"> "Hello! Sharing World"</h2>
+
+        <div class="header_part_text_div">
+          <h1 class="intro">Creative  Commons  global summit <br> 2023</h1>
+        </div>
+
+        <p class="about_header">
+          A joyful celebration believing in the value of
+          openness and sharing, creating a positive change
+          with people from over 80 countries is taking place
+          in October, in Korea.
+        </p>
+        <h2 class="date">
+          2023.10.15(THU) ~ 16(FRI)
+        </h2>
+        <h3 class="address">
+          @  National Museum of Korea, Art Center Nabi and more .
+        </h3>
+      </div>
+    </section>
+    <!-- main header end  -->
+    <!-- main section start  -->
+    <section class="main_section" id="program_section">
+
+      <div class="main_section_div">
+        <h3 class="main_header_text">Main Program</h3>
+        <div class="line"></div>
+
+        <Div class="main_cards_div">
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_lecture.svg" alt="Lecture">
+              <h3 class="main_card_title">Lecture</h3>
```

Comment:
- [ ] To ensure the cards' contents are perfectly centered within the main program section, I recommend centering the paragraphs too. Currently, it appears that the paragraphs **are not aligned in the center.** By making this adjustment, **you'll achieve a more balanced and visually appealing layout. 📐**
- [ ] Also, please check the `width` and `background color` of your cards to match the template.

   ### Your design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/eb05fc25-b73e-40bb-8de5-3154c9f25d55)

   ### Figma design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/7fec092d-e909-45e2-9bf6-40cef247e21c)